_data/site_menu.yml

```diff
@@ -118,6 +118,10 @@ using:
             path: "/docs/reference/latest/managing_providers/cloud_providers/openstack_providers.html"
             desc: "Managing OpenStack Providers"
 
+          - title: IBM CIC Providers
+            path: "/docs/reference/latest/managing_providers/cloud_providers/ibm_cic_providers.html"
+            desc: "Managing IBM CIC Providers"
+
```

Comment:
Does the order of these menu options matter?  It's not alphabetical but I can't tell how it's sorted.

Reply:
It is in the order of presentation in the UI - this should probably be added in alphabetically

Reply:
Yeah that's a good idea since they are all OpenStacks

Reply:
My only opinion is we should probably sort by name by category of provider.  It doesn't need to happen here but we should probably place this new one in the correct place going forward.